src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/performance-counter/table-performance-counter/table-performance-counter.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, Input, OnInit, TemplateRef, ViewChild } from '@angular/core';
+
+import { TablePerformanceCounterService } from '../services/table-performance-counter.service';
+
+/**
+ * Display the specified performance counters in a datatable.
+ */
+@Component({
+  selector: 'cd-table-performance-counter',
+  templateUrl: './table-performance-counter.component.html',
+  styleUrls: ['./table-performance-counter.component.scss']
+})
+export class TablePerformanceCounterComponent implements OnInit {
+
+  private columns: Array<object> = [];
+  private counters: Array<object> = [];
+
+  @ViewChild('valueTpl') public valueTpl: TemplateRef<any>;
+
+  /**
+   * The service type, e.g. 'rgw', 'mds', 'mon', 'osd', ...
+   */
+  @Input() serviceType: string;
+
+  /**
+   * The service identifier.
+   */
+  @Input() serviceId: string;
+
+  constructor(private performanceCounterService: TablePerformanceCounterService) { }
+
+  ngOnInit() {
+    this.columns = [
+      {
+        name: 'Name',
+        prop: 'name',
+        flexGrow: 1
+      },
+      {
+        name: 'Description',
+        prop: 'description',
+        flexGrow: 1
+      },
+      {
+        name: 'Value',
+        cellTemplate: this.valueTpl,
```

Comment:
Could you use "cellTransformation" instead?

Reply:
This is a very special cell template for the perf counter table, so we should not relocate it to the generic data table cell templates.